CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Added
 - More GPU information for debugging
+- Specify building as a single exe which relies on .NET Desktop Runtime by default in project file
```

Comment:
Replace with "Add missing project build settings" and author like the other contribuations

Reply:
Done